string category
int64 track_id
geometry_msgs/Point position
geometry_msgs/Point velocity
float64 confidence
